src/openzeppelin/utils/constants.cairo

```diff
@@ -41,12 +41,12 @@ const DEFAULT_ADMIN_ROLE: felt252 = 0;
 // Selectors
 //
 
-const OWNER_SELECTOR: felt252 = 0x2016836a56b71f0d02689e69e326f4f4c1b9057164ef592671cf0d37c8040c0;
-const TRANSFER_OWNERSHIP_SELECTOR: felt252 =
+const owner_SELECTOR: felt252 = 0x2016836a56b71f0d02689e69e326f4f4c1b9057164ef592671cf0d37c8040c0;
+const transfer_ownership_SELECTOR: felt252 =
```

Comment:
an alternative i'm just thinking about now is to have an `utils/selectors.cairo` module with all selectors in there without the `_SELECTOR` posfix, then we can

```cairo
use selectors;

call_contract_syscall(target, selectors::transfer_ownership, ...)
```
 
@ericnordelo @andrew-fleming

Reply:
+1 for `selectors::<method>`

Reply:
+1

Reply:
Done